BitcoinKitTests/PerformanceTests/MultiplicationTests.swift

```diff
@@ -0,0 +1,100 @@
+
+
+protocol ByteConvertible {}
+extension ByteConvertible {
+    func to32Bytes() -> [UInt8] {
+        let capacity = MemoryLayout<Self>.size
+        var mutableValue = self
+        var bytes = withUnsafePointer(to: &mutableValue) {
+
+            return $0.withMemoryRebound(to: UInt8.self, capacity: capacity) {
+
+                return Array(UnsafeBufferPointer(start: $0, count: capacity))
+            }
+        }
+        bytes = bytes.reversed()
+        while bytes.count < 32 {
+            bytes = [UInt8(0)] + bytes
+        }
+        return bytes
+    }
+}
+extension Int: ByteConvertible {}
+extension Int {
+    func toData() -> Data {
+        return Data(to32Bytes())
+    }
+}
+
+private func privateKeyWifFromInt(_ int: Int, network: Network = .mainnet) -> String {
```

Comment:
this was the only was I was able to perform point multiplication with low numbers `[1, 2, 3, 4,..., 999, 1000]` with the Curve order.